unittest/vslib/TestSwitchBCM56850.cpp

```diff
@@ -268,3 +268,135 @@ TEST(SwitchBCM56850, warm_update_queues)
 
     EXPECT_EQ(sw.get(SAI_OBJECT_TYPE_BRIDGE, sboid, 1, &attr), SAI_STATUS_SUCCESS);
 }
+
+TEST(SwitchBCM56850, test_tunnel_term_capability)
+{
+    auto sc = std::make_shared<SwitchConfig>(0, "");
+    auto signal = std::make_shared<Signal>();
+    auto eventQueue = std::make_shared<EventQueue>(signal);
+
+    sc->m_saiSwitchType = SAI_SWITCH_TYPE_NPU;
+    sc->m_switchType = SAI_VS_SWITCH_TYPE_BCM56850;
+    sc->m_bootType = SAI_VS_BOOT_TYPE_COLD;
+    sc->m_useTapDevice = false;
+    sc->m_laneMap = LaneMap::getDefaultLaneMap(0);
+    sc->m_eventQueue = eventQueue;
+
+    auto scc = std::make_shared<SwitchConfigContainer>();
+
+    scc->insert(sc);
+
+    SwitchBCM56850 sw(
+            0x2100000000,
+            std::make_shared<RealObjectIdManager>(0, scc),
+            sc);
+
+
+    sai_s32_list_t enum_val_cap;
+    int32_t list[2];
+    enum_val_cap.count = 2;
+    enum_val_cap.list = list;
+    EXPECT_EQ(sw.queryAttrEnumValuesCapability(0x2100000000, SAI_OBJECT_TYPE_TUNNEL,
```

Comment:
please use 1 parameter per line if you want to break function params to multiple lines, also do this across this PR

Reply:
Done